dragonfly/block/crop.go

```diff
@@ -0,0 +1,14 @@
+package block
+
+// Crop is an interface for all plants that are grown. A crop uses random chances on ticks to make sure that the crop should grow.
+// A crop doesn't necessarily have to be on a farmland block, as there are non-hydrated plants like trees and cactus that are planted on grass
```

Comment:
I wouldn't call saplings and cacti crops. I would only call plants that grow on farmland crops.